docs/guides/serverless.md

```diff
@@ -0,0 +1,201 @@
+# Using AnyCable to power serverless Node.js applications
+
+AnyCable is a great companion for your serverless Node.js applications needing real-time features. It can be used as a real-time server with no strings attached: no vendor lock-in, no microservices spaghetti, no unexpected PaaS bills. Keep your logic in one place (your JS application) and let AnyCable handle the low-level stuff.
+
+## Overview
+
+To use AnyCable with a serverless Node.js application, you need to:
+
+- Deploy AnyCable-Go to a platform of your choice (see [below](#deploying-anycable-go)).
+- Configure AnyCable API handler in your JS application.
+
+<picture class="captioned-figure">
+     <source srcset="/assets/serverless-dark.png" media="(prefers-color-scheme: dark)">
+     <img align="center" alt="AnyCable + Node.js serverless architecture" style="max-width:80%" title="AnyCable + Node.js serverless architecture" src="/assets/serverless-light.png">
+</picture>
+
+AnyCable-Go will handle WebSocket connections and translate incoming commands into API calls to your serverless functions, where you can manage subscriptions and respond to commands. The client-server communication is expected to use [Action Cable protocol](../misc/action_cable_protocol.md).
```

Comment:
I would probably remove "The client-server communication is expected to use [Action Cable protocol](../misc/action_cable_protocol.md)." because it is sort of excessive knowledge, it seems.